src/db/migrations/20200418082538-add-user-civility-display-email.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  QueryInterface,
+  DataTypes,
+} from "sequelize";
+
+export = {
+  up: (queryInterface: QueryInterface, Sequelize) => queryInterface.addColumn("users", "civility", {
+    type: new DataTypes.ENUM("Ms.", "Mr.", "Dr."),
```

Comment:
define somewhere and import (same that you did with contacts)

Reply:
In case we want to add/remove/update the enum, makes it easier

Reply:
No, I did it on purpose, if we add / remove a field to the enum, we have to keep track of what was added / removed !!!